src/foam/u2/view/ChoiceView.js

```diff
@@ -245,7 +248,10 @@ foam.CLASS({
             .end();
         }
 
-        return self.E().translate(self.text + ".name", self.text);
```

Comment:
Actually, just revert back to this line but add 'text' as a parameter on line 229.